pkg/logs/consumer/minio.go

```diff
@@ -0,0 +1,102 @@
+package consumer
```

Comment:
Can't we init minio Writer on the beginning and just push chunk of data to it. 

This implementation be quite slow - you're loading appending removing object in one go - it'll be impossible to write even 100 lines per second